ph5/clients/ph5tostationxml.py

```diff
@@ -296,12 +302,9 @@ def __init__(self, manager):
         self.total_number_stations = 0
         self.unique_errors = set()
         self.checked_data_files = {}
-        self.unique_filenames_n_i = []
         self.manager.ph5.read_response_t()
         validation.check_resp_unique_n_i(
             self.manager.ph5, self.unique_errors, None)
-        self.resp_load_already = validation.check_resp_load(
-            self.manager.ph5.Response_t, self.unique_errors, None)
```

Comment:
Where has this been moved? It seems that validation.check_resp_load has been updated and replaced. What function replaced it?

Reply:
Function check_resp_load() has been changed to check_response_filename() and has been moved to Lines 674-678 inside read_networks(). I think it's better not to check response in __init__()

Reply:
Function check_resp_load() has been changed to check_response_filename() at https://github.com/PIC-IRIS/PH5/blob/ccf4017bd9eb8dd327a4bdf5d4dfe65802841288/ph5/clients/ph5tostationxml.py#L674-L678 in read

I think it's better not to check response error in __init__().

Reply:
Thanks for this clarification.